lib/dash/dash_parser.js

```diff
@@ -383,6 +383,17 @@ shaka.dash.DashParser = class {
     }
 
     // The segments are available earlier than the availability start time.
+    // If the stream is low latency and the user has not configured the
+    // lowLatencyMode, but if it has been configured to activate the
+    // llowLatencyMode if a stream of this type is detected, we automatically
```

Comment:
llowLatencyMode => autoLowLatencyMode( I guess, also llowLatencyMode is misspelled )

Reply:
Changed! Thanks